test/distributed.in.cpp

```diff
@@ -140,3 +143,40 @@ TEST(distributed_${BACKEND_NAME}, broadcastdistributed)
         EXPECT_EQ(v, read_vector<float>(result));
     }
 }
+
+//MLSL does not support send recv
+#if !defined(NGRAPH_DISTRIBUTED_MLSL_ENABLE)
+NGRAPH_TEST(distributed_${BACKEND_NAME}, send_recv)
+{
+    auto shape = Shape{2, 2};
+    auto A = make_shared<op::Parameter>(element::f32, shape);
+    auto comm_size = get_distributed_interface()->get_size();
+    if (comm_size != 2)
+    {
+        return;
+    }
+    auto rank = get_distributed_interface()->get_rank();
+    std::shared_ptr<Function> f;
+    if (rank == 0)
+    {
+        f = make_shared<Function>(make_shared<op::Send>(A, 1), ParameterVector{A});
```

Comment:
This test seems that only one process sends and one processes receives. But for real application, it could be all processes send and then all processes receive. Could we add some tests for that scenario?

Reply:
add a ring send recv test